ansible/roles/operator-pipeline/templates/openshift/tasks/get-ci-results-attempt.yml

```diff
@@ -9,16 +9,59 @@ spec:
     - name: pyxis_url
     - name: cert_project_id
     - name: bundle_version
+    - name: operator_name
   results:
     - name: preflight_results_exists
+    - name: test_result_id
+    - name: test_logs_id
```

Comment:
Why do we need to get the log? I don't believe we need this in the hosted pipeline.